### PR TITLE
Adding to the Pool

### PR DESCRIPTION
**Pull request by Tornyenyo**
Let see how this collaboration goes.

### CODE REVIEW

File: C_tasks-here/bentii/C_codes-task4.c [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+	printf("Enter the second number:");
+
+	scanf("%d", &secondNumber);
+
+	sumOfNumber = firstNumber + secondNumber;
+
+	printf("The total of the numbers is: %d",sumOfNumber);
+
+	return 0
```
Comment (RexGhub) [inline C_tasks-here/bentii/C_codes-task4.c]: you forgot to add semi colon after 0. i.e return 0;
Kindly fix

File: C_tasks-here/bentii/C_codes-task4.c [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+int main(){
+int sumOfNumber, firstNumber, secondNumber
+;
```
Comment (RexGhub) [inline C_tasks-here/bentii/C_codes-task4.c]: correct this. it should be at the end of line 3. i.e. after secondNumber

**Review by RexGhub — CHANGES_REQUESTED** [review]
Fix these changes in the comments then create a new pull request

File: C_tasks-here/bentii/C_codes-task4.c (hunk 1 above)
Comment (Tornyenyo) [reply C_tasks-here/bentii/C_codes-task4.c]: okay...

File: C_tasks-here/bentii/C_codes-task4.c (hunk 2 above)
Comment (Tornyenyo) [reply C_tasks-here/bentii/C_codes-task4.c]: ok

**Review by RexGhub — COMMENTED** [review]
Review complete. Well done

**Review by RexGhub — APPROVED** [review]

**Pull request merged**